plugins/rbac-backend/openapi.yaml

```diff
@@ -0,0 +1,636 @@
+    put:
+      description: Updates one or more permission policies for a specified entity.
+      parameters:
+        - $ref: '#/components/parameters/nameParam'
+        - $ref: '#/components/parameters/namespaceParam'
+        - $ref: '#/components/parameters/kindParam'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                oldPolicy:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      permission:
+                        type: string
+                        description: >-
+                          Permission from a specific plugin, Resource type or
+                          name
+                      policy:
+                        type: string
+                        description: >-
+                          Policy action for the permission: create, read,
+                          update, delete, use
+                      effect:
+                        type: string
+                        description: allow or deny
+                newPolicy:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      permission:
+                        type: string
+                        description: >-
+                          Permission from a specific plugin, Resource type or
+                          name
+                      policy:
+                        type: string
+                        description: >-
+                          Policy action for the permission: create, read,
+                          update, delete, use
+                      effect:
+                        type: string
+                        description: allow or deny
+      responses:
+        '200':
+          description: Request was successful.
+        '400':
+          description: Input Error
+        '403':
+          description: Refusal to authorize
+    delete:
+      description: >-
+        Deletes a permission policy or a group of permission policies of a
+        specified entity.
+      parameters:
+        - $ref: '#/components/parameters/nameParam'
+        - $ref: '#/components/parameters/namespaceParam'
+        - $ref: '#/components/parameters/kindParam'
+        - name: permission
+          in: query
+          description: Permission from a specific plugin, Resource type or name
+          required: false
+          schema:
+            type: string
+        - name: policy
+          in: query
+          description: 'Policy action for the permission: create, read, update, delete, use'
+          required: false
+          schema:
+            type: string
+        - name: effect
+          in: query
+          description: allow or deny
+          required: false
+          schema:
+            type: string
```

Comment:
I don't see these query params for delete method:
```suggestion
```

Reply:
I saw them listed here: https://github.com/janus-idp/backstage-plugins/blob/main/plugins/rbac-backend/docs/apis.md#delete-permission

Reply:
I see, but this information is outdated. Can you fix also corresponding docs?